src/cace/ari/base.c

```diff
@@ -215,6 +215,11 @@ void ari_objpath_copy(ari_objpath_t *obj, const ari_objpath_t *src)
     obj->ari_type     = src->ari_type;
 }
 
+bool ari_valid_type_for_objpath(ari_type_t type)
```

Comment:
Very minor feedback; if this is a local-only function (no header declaration) mark it `static` to inform the compiler.

Reply:
Done, thanks @BrianSipos!